modules/imgimport/test/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+enable_testing()
+
+if(Boost_FOUND AND OpenCV_FOUND)
+    add_definitions("-DBOOST_LOG_DYN_LINK")
+    add_executable(picture_test picture_test.cpp)
+    add_executable(csv_test csv_test.cpp)
+    target_link_libraries(picture_test ${OpenCV_LIBS} ${Boost_LIBRARIES} Core ImageImport)
+    target_link_libraries(csv_test ${OpenCV_LIBS} ${Boost_LIBRARIES} Core ImageImport)
+
+    # Tests
+    add_test("picture_test" picture_test ${TESTDATA_DIR} ${TESTDATA_DIR}/pictureimport_test_csv.csv --log_format=XML --log_sink=PICTURE_TEST-boost.xml --log_level=all --report_level=no)
+    add_test("csv_test" csv_test ${TESTDATA_DIR} ${TESTDATA_DIR}/pictureimport_test_csv.csv --log_format=XML --log_sink=CSV_TEST-boost.xml --log_level=all --report_level=no)
```

Comment:
When I said that the names have the format TEST*.xml I meant that they need to start with the prefix "TEST" , i.e. TEST-csv.xml instead of CSV_TEST-boost.xml (notice that Jenkins now reports only running one test when previously it was running two). 
The part which currently says boost is what should be unique, the boost suffix was actually left over from when I was originally testing the boost test output format and I forgot to change it when writing the target identification test.